pdf/model/writer.go

```diff
@@ -93,8 +158,18 @@ func NewPdfWriter() PdfWriter {
 
 	// Creation info.
 	infoDict := MakeDict()
-	infoDict.Set("Producer", MakeString(getPdfProducer()))
```

Comment:
Could change the order of the statements for a bit more structure.
Also, maybe some of the fields should be added only if they are not empty.

```
	infoDict.Set("Producer", MakeString(getPdfProducer()))
	infoDict.Set("Creator", MakeString(getPdfCreator()))
        infoDict.Set("Author", MakeString(getPdfAuthor()))
	infoDict.Set("Subject", MakeString(getPdfSubject()))
	infoDict.Set("Title", MakeString(getPdfTitle()))
	infoDict.Set("Keywords", MakeString(getPdfKeywords()))

        // Set creation and modified dates.
	if cd, err := NewPdfDateFromTime(getPdfCreationDate()); err == nil {
		infoDict.Set("CreationDate", cd.ToPdfObject())
	}
	if cd, err := NewPdfDateFromTime(getPdfModifiedDate()); err == nil {
		infoDict.Set("ModDate", cd.ToPdfObject())
	}
```